app.js

```diff
@@ -34,6 +37,9 @@ const responseTime = async (ctx, next) => {
     ctx.set('X-Response-Time', `${ms}ms`)
 }
 
+// upload file middleware
+const uploadMiddleware = multer().fields([{ name: 'file', maxCount: 1 }])
+
 app.use(error_handler, loggerMid, responseTime)
```

Comment:
error_handler: check naming convetion

Reply:
loggerMid: naming convention, shoud be a verb and should not use short name 'Mid'
responseTime: naming convention, should be a verb